trypurescript.cabal

```diff
@@ -20,7 +20,7 @@ executable trypurescript
                    filepath -any,
                    Glob -any,
                    scotty -any,
-                   purescript ==0.11.6,
```

Comment:
Could you please update the version of the package itself as well?

Reply:
On line 2, I mean.

Reply:
To 0.11.6 as well?

Reply:
Yes please

Reply:
Done!